component/block_cache/block_cache.go

```diff
@@ -1258,7 +1268,7 @@ func (bc *BlockCache) diskEvict(node *list.Element) {
 // checkDiskUsage : Callback to check usage of disk and decide whether eviction is needed
 func (bc *BlockCache) checkDiskUsage() bool {
 	data, _ := common.GetUsage(bc.tmpPath)
-	usage := uint32((data * 100) / float64(bc.diskSize))
+	usage := uint32((data * 100) / float64(bc.diskSize/_1MB))
```

Comment:
why are we changing this method here ? disk size is in MB and data is also in MB right?

Reply:
For mem-size and block-size the values are saved in bytes but for disk-size the value was saved in MB so I changed it to bytes to make the output uniform. Hence here I need to convert bytes to MB for the test case